src/rust/plugin-registry/src/nomad_cli.rs

```diff
@@ -0,0 +1,62 @@
+use std::{
+    collections::HashMap,
+    io::Write,
+    process::Command,
+};
+
+use nomad_client_gen::models;
+use tempfile::NamedTempFile;
+
+#[derive(Debug, thiserror::Error)]
+pub enum ParseHclError {
+    #[error("IOError {0:?}")]
+    IOError(#[from] std::io::Error),
+    #[error("DeserializeJsonError")]
+    DeserializeJsonError(#[from] serde_json::Error),
+}
+
+pub type NomadVars = HashMap<String, String>;
+
+/// Convert HCL2 + variables into JSON
+fn parse_hcl2_to_json(hcl2: &str, vars: HashMap<String, String>) -> Result<String, std::io::Error> {
```

Comment:
Looks like an area where you'd want to use your NomadVars type alias.

Reply:
done